app/controllers/rating_questions_controller.rb

```diff
@@ -1,27 +1,84 @@
 class RatingQuestionsController < ApplicationController
+  
+  before_action :find_question, only: [:show, :destroy, :edit, :update]
+  
   def index
     @rating_questions = RatingQuestion.all
+
+    # binding.pry
   end
 
   def new
     @rating_question = RatingQuestion.new
   end
 
   def create
-    @rating_question = RatingQuestion.new(question_params)
+    head 400 if !rating_question_params[:title]
```

Comment:
`strong_parameters` will catch this for you and return the same status (400). You do not need this line here in the controller. However, removing it might cause some of your tests to fail -- they will likely raise an `ActionController::ParameterMissing` exception. See if you can solve these tests.